src/test/ratings.spec.js

```diff
@@ -0,0 +1,51 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('Testing Rating of articles', () => {
+  it('it should return a 400 status code if the request is bad', done => {
+    chai
+      .request(app)
+      .patch('/api/v1/articles/:articleId/:ratings')
+      .send({})
+      .end((error, response) => {
+        expect(response.status).to.equal(400);
+        done();
+      });
+  });
+  it('it should return an relevant error message if the request is bad', done => {
+    chai
+      .request(app)
+      .patch(`/api/v1/articles/:articles/:ratings`)
+      .end((error, response) => {
+        response.body.message.should.be.a('string');
+        done();
+      });
+  });
+  it('it should return status 200 if the request was successful', done => {
+    chai
+      .request(app)
+      .patch(`/api/v1/articles/${1}/${4}`)
+      .set('x-access-token', process.env.TEST_TOKEN)
+      .end((error, response) => {
+        expect(response.status).to.equal(200);
+        expect(response.body.data.ratings).to.equal('4');
+        expect(response.body.data.articleId).to.equal(1);
+        expect(response.body.data.userId).to.equal(2);
```

Comment:
Why are we returning the `userId` after rating? What `userId` is this? The person who rated or the author of the article?
In either case, it provides no added value.